spec/custom_html_renderer_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require_relative '../lib/custom_html_renderer'
+
+RSpec.describe CustomHTMLRenderer do
```

Comment:
TBH I'm not entirely sure of the usefulness of these tests – open to suggestions.